reporters/GlobalSummaryReporter.js

```diff
@@ -483,6 +494,22 @@ class GlobalSummaryReporter {
     const blocksToLookBack = Math.ceil(lookbackTimeInSeconds / blockTimeInSeconds);
     return blocksToLookBack;
   };
+
+  _getTokenHolders = async () => {
+    // TODO: This is a fragile implementation that scrapes etherscan's token holder page. It would likely fail if the
+    // the etherscan HTML document changes.
+    const etherscanTokenHoldersUrl =
+      "https://etherscan.io/token/generic-tokenholders2?a=0x6d002a834480367fb1a1dc5f47e82fde39ec2c42&s=2004251000000000000000000";
```

Comment:
Shouldn't we pull the token address from an environment variable or something? You could also take advantage of the `getUniswapPairDetails` method in #1576 to get the pool address for a particular EMP if that's helpful.

Reply:
yes we should do this. We can just grab `emp.syntheticCurrency.address`